README.md

```diff
@@ -7,7 +7,8 @@
 ## AutoML for Image, Text, and Tabular Data
 
 [![Latest Release](https://img.shields.io/github/v/release/awslabs/autogluon)](https://github.com/awslabs/autogluon/releases)
-[![Build Status](https://ci.gluon.ai/view/all/job/autogluon/job/master/badge/icon)](https://ci.gluon.ai/view/all/job/autogluon/job/master/)
+[![Continuous Integration](https://github.com/awslabs/autogluon/actions/workflows/continuous_integration.yml/badge.svg)](https://github.com/awslabs/autogluon/actions/workflows/continuous_integration.yml)
+[![Platform Tests](https://github.com/awslabs/autogluon/actions/workflows/platform_tests-command.yml/badge.svg?event=schedule)](https://github.com/awslabs/autogluon/actions/workflows/platform_tests-command.yml)
```

Comment:
Do you know what the `event=schedule` query parameter means on the badge? I loaded the badge with and without it and didn't notice any difference.

Reply:
Sort of answering my own question, I found [this documentation](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge#using-the-event-parameter)

Reply:
Oh yea, for our platform tests we have different triggers. The schedule one is the one that runs every night. We want to show that.

Reply:
This badge displays the status of workflow runs triggered by the `schedule` event. Open question here if that's the right event. [Examples](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge#using-the-event-parameter) are using `push`.

Reply:
@gradientsky it is the right event, as we want to display the status of the nightly tests not status of results from some manual trigger in a PR